sw/device/tests/sram_ctrl_smoketest.c

```diff
@@ -0,0 +1,88 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_sram_ctrl.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/check.h"
+#include "sw/device/lib/testing/test_framework/test_main.h"
```

Comment:
Now that all tests have migrated to the OTTF (in #9368) could you change this to:

```c
#include "sw/device/lib/testing/test_framework/ottf.h"
```

The old `test_main` framework will be deprecated shortly.

Reply:
Changes done and pushed. Thanks Tim.